lerna-script/test/iterators.spec.js

```diff
@@ -7,11 +7,8 @@ const {expect} = require('chai'),
   invertPromise = require('invert-promise')
 
 describe('iterators', () => {
-
-  ['forEach', 'parallel', 'batched'].forEach(type => {
-
+  ;['forEach', 'parallel', 'batched'].forEach(type => {
```

Comment:
why `;`?

Reply:
TLDR that is a prettier behavior with `"semi": false` option

Was raised several times in the prettier repository (the catalog of that questions is listed here: https://github.com/prettier/prettier/issues/1935#issuecomment-306729468)

Reply:
So I just manually triggered `run prettier --write .` with `.prettierrc` from master and decrease amount of changes with additional:
```
  "trailingComma": "none",
  "arrowParens": "avoid",
```